inc/deprecated.php

```diff
@@ -31,3 +31,16 @@ function siteorigin_corp_excerpt_more( $more ) {
 }
 endif;
 add_filter( 'excerpt_more', 'siteorigin_corp_excerpt_more' );
+
+if ( ! function_exists( 'siteorigin_corp_woocommerce_archive_product_image_buttons' ) ) :
+/**
+ * Archive product image buttons.
+ */
+function siteorigin_corp_woocommerce_archive_product_image_buttons() { ?>
+	<?php if ( siteorigin_setting( 'woocommerce_quick_view' ) ) {
```

Comment:
This is very minor but is there a specific reason you've added closing and opening tags for PHP here?

Reply:
Never mind. Just noticed this was already the case.